src/cbmc/fault_localization.cpp

```diff
@@ -192,6 +194,37 @@ void fault_localizationt::report(irep_idt goal_id)
                    << eom;
 }
 
+void fault_localizationt::report_xml(irep_idt goal_id, xmlt& xml)
+{
```

Comment:
That function should be rewritten to return XML node, rather than accept it as a modifiable reference.